src/pretix/plugins/sendmail/tasks.py

```diff
@@ -36,6 +39,9 @@ def send_mails(event: Event, user: int, subject: dict, message: dict, orders: li
                 if p.item_id not in items and not any(a.item_id in items for a in p.addons.all()):
                     continue
 
+                if forms.MailForm.NOT_CHECKED_IN not in checkin_lists and not any(str(c.list_id) in checkin_lists for c in p.checkins.all()):
+                    continue
```

Comment:
The logic seems off here.

If I only select "Not checked in", it does send emails to everyone, even those who checked in, doesn't it?